packages/cms-cli/commands/auth.js

```diff
@@ -48,13 +48,13 @@ const SUPPORTED_AUTHENTICATION_PROTOCOLS_TEXT = commaSeparatedValues(
   ALLOWED_AUTH_METHODS
 );
 
-async function authAction(type, options) {
```

Comment:
This is fine, but just fyi the reason I had changed it was because to me `command` may have a connotation of using `commander`, whereas with `yargs` I believe it is actually the options that are being passed

Reply:
Gotcha. I don't really care if we use `command` or `options` for this, to me having consistency is the most important thing. It was confusing when looking at different commands to see options be different things at times.

Reply:
After putting more thought into this, I think `options` makes more sense because it could be either `Commander.command` or `object`(Yargs options) types. Updated them to `options` throughout.